src/main/scala/castalia/StubConfigParser.scala

```diff
@@ -1,15 +1,36 @@
 package castalia
 
-import castalia.model.Model.StubConfig
+import castalia.model.Model.{ResponseConfig, DefaultResponseConfig, StubConfig}
 
 
 object StubConfigParser {
 
   def parseStubConfig(jsonFile: String): StubConfig = {
-    JsonConverter.parseJson[StubConfig](jsonFile)
+    val initialConfig = JsonConverter.parseJson[StubConfig](jsonFile)
+    (initialConfig.default) match {
+      case (Some(default)) => StubConfig( initialConfig.endpoint, initialConfig.default, addDefaults(default, initialConfig.responses))

```

Comment:
something like `initialConfig.copy(defaults = addDefaults(default, initialConfig.responses))` should do the same